sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -274,30 +294,14 @@ def __init__(self, request: HttpRequest,
         self.headers = CIMultiDict(aiohttp_response.headers)
         self.reason = aiohttp_response.reason
         self.content_type = aiohttp_response.headers.get('content-type')
-        self._body = None
-        self._decompressed_body = None
+        self._content = None
```

Comment:
changed the name of the old response's private internal variable from `_body` to `_content`. The reason is with the backcompat mixins, we're adding the private method `_body`, which conflates with the private ivar